build.sbt

```diff
@@ -19,5 +19,4 @@ libraryDependencies ++= Seq(
 )
 
 
-fork in run := true
```

Comment:
Oh, I meant that I removed it so that it would start server properly, so I wasn't sure if that was the right choice.